src/pages/TravelBlog/HowItWorks/HowItWorks.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from "react";
+import BlogHeader from "../../../components/TravelBlog/sectionHeader/SectionHeader";
+import Section from "../../../components/TravelBlog/section/Section";
+import Grid from "../../../components/TravelBlog/grid/Grid";
+import Todo from "../../../components/TravelBlog/todo/Todo";
+
+function HowItWorks() {
+  const [smallScreen, setSmallScreen] = useState(false);
+
+  useEffect(() => {
+    window.addEventListener("resize", checkScreen);
+    checkScreen();
+
+    return () => window.removeEventListener("resize", checkScreen);
+  }, []);
+
+  const checkScreen = () => {
+    window.innerWidth <= 760 ? setSmallScreen(true) : setSmallScreen(false);
```

Comment:
This logic in completely duplicated can we find a way to share it with the other component where it’s used?